modules/digispark/module.js

```diff
@@ -1,12 +1,6 @@
 /* jshint laxbreak: true */
 
 new Object({
-  moduleMeta: { /* Module infos */
-    displayName: 'Digispark',
-    version: 1.0,
-    author: '@alcyonsecurity / NIXU'
```

Comment:
Should we keep these credits here?

Reply:
Ah never mind.

Reply:
Yes, for consistency I think that's great, and for scallability too :D

Reply:
They are in the `module.meta` files now ^^

Reply:
Yup I saw that.